pages/docs/how-to-guides/links.mdx

```diff
@@ -56,10 +58,56 @@ To add more links add another object inside the links collection. For example:
 
 _Clicks on these social shortcuts that match any of your links will also increment their respective counters_
 
-### What Links with grouping looks like
+### Grouping your Links (optional)
+To place your social links into categories, you can use names like `Business`,  `Socials Primary`, or `:sparkles: Personal` in the `group` string.
 
-![LinkFree profile with grouped links](https://user-images.githubusercontent.com/624760/214538171-099cac71-d2f7-49a8-b3bb-2747506a1bac.png)
+These can include any alphanumeric, unicode characters and symbols.
 
+<ClipboardCopy>
+```js
+"links": [
+ {
+      "group": "Business",
+      "name": "Business: DevRel Services",
+      "url": "http://eddiejaoude.io",
+      "icon": "FaLink"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/eddiejaoude",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "YouTube: Learn more about Open Source",
+      "url": "https://youtube.com/eddiejaoude",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "GitHub: Collaborate on Open Source",
+      "url": "https://github.com/eddiejaoude",
+      "icon": "FaGithub"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "LinkedIn: Let's connect",
+      "url": "https://linkedin.com/in/eddiejaoude",
+      "icon": "FaLinkedin"
+    },
+
+]
+   
+  ```
+</ClipboardCopy>
+
+Here's how the categories look on our website:
+![Group-categories](https://github.com/EddieHubCommunity/LinkFree/assets/105683440/813f1706-b4f9-4ff3-be2b-0a7a4954737b)
+Here's how they look with the complete profile: 
+![LinkFree profile with grouped links](https://user-images.githubusercontent.com/624760/214538171-099cac71-d2f7-49a8-b3bb-2747506a1bac.png)
```

Comment:
I've added a line break here between the Markdown and Javascript which will fix the build failure:

```suggestion
![LinkFree profile with grouped links](https://user-images.githubusercontent.com/624760/214538171-099cac71-d2f7-49a8-b3bb-2747506a1bac.png)

```